chrome/content/zotero/preferences/quickCopySiteEditor.xul

```diff
@@ -46,6 +47,7 @@
 				var io = window.arguments[0];
 				io.domain = document.getElementById('zotero-quickCopy-domain').value;
 				io.format = document.getElementById('zotero-quickCopy-menu').value;
+				io.locale = document.getElementById('locale-menu').label !== '' ? document.getElementById('locale-menu').value : '';
```

Comment:
My guess was going to be that you're checking if the locale menu is disabled (because of default-locale style), in which case it wouldn't make sense to use that locale value for quick copy. I'm actually not sure if this is what this checks for (or even if this would work to check what I said).